tests/one_time_keys_test.go

```
@@ -21,19 +21,38 @@ import (
+	var result gjson.Result
+	claimer.MustDo(t, "POST", []string{
 		"_matrix", "client", "v3", "keys", "claim",
 	}, client.WithJSONBody(t, map[string]any{
 		"one_time_keys": map[string]any{
 			target.UserID: map[string]any{
 				target.DeviceID: "signed_curve25519",
 			},
 		},
+	}), client.WithRetryUntil(10*time.Second, func(res *http.Response) bool {
+		result = must.ParseJSON(t, res.Body)
+		res.Body.Close()
```

golangci-lint (2.12.2)

[error] Error return value of `res.Body.Close` is not checked

(errcheck)